yak/CMakeLists.txt

```diff
@@ -190,7 +194,7 @@ target_include_directories(${PROJECT_NAME}_marching_cubes PUBLIC
 target_link_libraries(${PROJECT_NAME}_marching_cubes
     PUBLIC
   ${PROJECT_NAME}
-  OpenMP::OpenMP_CXX
+  ${OpenMP_CXX_LIBRARIES}
```

Comment:
Recommend creating an imported target. This should also be added to your cmake config for this package.

```

find_package(OpenMP REQUIRED)

if(NOT TARGET OpenMP::OpenMP_CXX)
    find_package(Threads REQUIRED)
    add_library(OpenMP::OpenMP_CXX IMPORTED INTERFACE)
    set_property(TARGET OpenMP::OpenMP_CXX
                 PROPERTY INTERFACE_COMPILE_OPTIONS ${OpenMP_CXX_FLAGS})
    # Only works if the same flag is passed to the linker; use CMake 3.9+ otherwise (Intel, AppleClang)
    set_property(TARGET OpenMP::OpenMP_CXX
                 PROPERTY INTERFACE_LINK_LIBRARIES ${OpenMP_CXX_FLAGS} Threads::Threads)

endif()
```

Reply:
I added the suggestion function to CMakeLists.txt. What should I add to the package CMake config?